.gitignore

```diff
@@ -1 +1,4 @@
 vendor/bundle
+
+.DS_Store
```

Comment:
You are missing changes to `k8s.yml`:
https://github.com/okteto/go-getting-started/pull/10/files#diff-9aa5c3faef5c8a780db30dc0dd41b946e5732af5677ee2fc17d3db774fe1a57b
You are also missing to commit the `.stignore` file.

Reply:
there was an existing .stignore in this repo, should i rename/delete it and run up to generate a new one or?